server/views/applications/delete.njk

```diff
@@ -0,0 +1,61 @@
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+{% from "govuk/components/panel/macro.njk" import govukPanel %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "../components/formFields/form-page-radios/macro.njk" import formPageRadios %}
+{% from "../partials/showErrorSummary.njk" import showErrorSummary %}
+
+{% extends "../partials/layout.njk" %}
+
+{% set mainClasses = "app-container govuk-body" %}
+{% set name = personName(application.person, 'Limited access offender') %}
```

Comment:
we dont need to do this because referrers can only add/edit draft applications which they are authorised to view the offenders details so they should not be able to access this part of the journey